EasyRestProjectNetTeam2/EasyRestPages/ManageMenuPage.cs

```diff
@@ -1,14 +1,16 @@
-﻿using OpenQA.Selenium;
+﻿using EasyRestProjectNetTeam2.EasyRestComponentsObj;
+using OpenQA.Selenium;
 using SeleniumExtras.PageObjects;
 using System.Collections.Generic;
 
 namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class ManageMenuPage : BasePage
     {
+        public LeftBarComponent LeftBarComponent { get; set; }
```

Comment:
don't use setter for components because someone might re-initialize the component